File: packages/glass-blur/ios/ProgressiveBlurViewManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(ProgressiveBlurViewManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(startBlur, NSNumber)
RCT_EXPORT_VIEW_PROPERTY(endBlur, NSNumber)
@end

File: packages/glass-blur/ios/ProgressiveBlurViewManager.swift
import Foundation
import React

@objc(ProgressiveBlurViewManager)
class ProgressiveBlurViewManager: RCTViewManager {
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }

    override func view() -> UIView! {
        return ProgressiveBlurView()
    }
}
